src/app/pages/login/login.page.ts

```diff
@@ -45,4 +57,8 @@ export class LoginPage implements OnInit {
     return this.signInForm.controls;
   }
 
+  get formErrors() {
+    return this.signInForm.errors;
```

Comment:
Again, adding errors directly to the form group works better than trying to add an error to an existing or new form control manually.